python/tutorials/08-grouped-gemm.py

```diff
@@ -222,6 +427,17 @@ def triton_perf_fn(a_ptrs, b_ptrs, c_ptrs, sizes, lds, group_size):
     )
 
 
+def triton_tma_perf_fn(a_ptrs, b_ptrs, c_ptrs, sizes, lds, group_size, dtype):
+    grid = lambda META: (META['NUM_SM'], )
+    grouped_matmul_tma_kernel[grid](a_ptrs, b_ptrs, c_ptrs, sizes, lds, group_size, FP8=torch.float8_e4m3fn == dtype,
+                                    NUM_SM=num_sms())
+    #num_warps = 8,
+    #num_stages = 4 if torch.float8_e4m3fn == dtype else 3,
+    #BLOCK_SIZE_M = 128,
+    #BLOCK_SIZE_N = 256,
+    #BLOCK_SIZE_K = 128 if torch.float8_e4m3fn == dtype else 64,
```

Comment:
is this there on purpose? I would suggest removing it or adding a comment if there is a special meaning

Reply:
That was left over from some testing. I just removed it.